src/common/conf_test.cpp

```diff
@@ -752,3 +752,27 @@ TEST(ConfTests, ProxyEnvironmentVariables) {
 		EXPECT_FALSE(result);
 	}
 }
+
+TEST(ConfTests, FallbackConfig) {
+	mtesting::TemporaryDirectory tmpdir;
+
+	string conf_file = path::Join(tmpdir.Path(), "mender.conf");
+	{
+		ofstream f(conf_file);
+		f << R"({"ServerURL": "https://right-server.com"})";
```

Comment:
Example.com? :laughing:

Reply:
I guess that's the go-to choice! But we don't connect to it anyway, it's just parsing.